src/QMCDrivers/QMCDriverNew.cpp

```diff
@@ -61,10 +64,39 @@ QMCDriverNew::QMCDriverNew(const ProjectData& project_data,
       project_data_(project_data),
       setNonLocalMoveHandler_(snlm_handler)
 {
-  //create and initialize estimator
-  estimator_manager_ = std::make_unique<EstimatorManagerNew>(population_.get_golden_hamiltonian(), myComm);
+  using DV = ProjectData::DriverVersion;
+  switch (project_data_.get_driver_version())
+  {
+  case DV::BATCH: {
+    // This is done so that the application level input structures reflect the actual input to the code.
+    // While the actual simulation objects still take singular input structures at construction.
+    auto makeEstimatorManagerInput = [](auto& global_emi, auto& local_emi) -> EstimatorManagerInput {
+      if (global_emi.has_value() && local_emi.has_value())
+        return {global_emi.value(), local_emi.value()};
+      else if (global_emi.has_value())
+        return {global_emi.value()};
+      else if (local_emi.has_value())
+        return {local_emi.value()};
+      else
+        return {};
+    };
+
+    estimator_manager_ =
+        std::make_unique<EstimatorManagerNew>(comm,
+                                              makeEstimatorManagerInput(global_emi,
+                                                                        qmcdriver_input_.get_estimator_manager_input()),
+                                              population_.get_golden_hamiltonian(), *population.get_golden_electrons(),
+                                              population.get_golden_twf());
+  }
+  break;
+  case DV::LEGACY:
+    // This estimator still requires its put be called later
+    estimator_manager_ = std::make_unique<EstimatorManagerNew>(population_.get_golden_hamiltonian(), myComm);
+    break;
+  }
```

Comment:
I feel this switch very very strange. QMCDriverNew is the base class only used by batched drivers. Why DV::LEGACY matters?
I think this part of code merging global and local estimators should be moved to QMCDriverFactory::createQMCDriver before drivers are instantiated.

Reply:
This is to support the vmcbatch, dmcbatch style input.  I'd much prefer it be temporary.

Reply:
vmcbatch and dmcbatch should be all aligned to "batched driver" input style. I'm happy with a breaking change.

Reply:
Parsing needs to know its version batched from the start. It needs to know the new estimators blocks are parsed, and we'll need to halt on the hamiltonian containing legacy estimator input or any other nonsense input.  We can't reparse nodes outside of the qmc section node based on the qmc section nodes attributes.
I think we should just remove vmcbatch and dmcbatch as section types.

Reply:
vmcbatch and dmcbatch are just section type names, the underlying object types are DMCBatched and VMCBatched. For them, I mean there is no need of supporting legacy estimators. What I was requesting is as simple as deleting all `DV::LEGACY` code paths in `QMCDriverNew.cpp`

Reply:
This will actually work I did actually find a way to make vmcbatch and dmcbatch still work by always looking for an reading if there any global <estimators> section.